src/gen_nb_server.erl

```
@@ -111,7 +125,7 @@ handle_info({inet_async, ListSock, _Ref, {ok, CliSocket}}, #state{cb=Callback, s
   inet_db:register_socket(CliSocket, inet_tcp),
   case Callback:new_connection(CliSocket, ServerState) of
     {ok, NewServerState} ->
-      prim_inet:async_accept(ListSock, -1),
+      {ok, _} = prim_inet:async_accept(ListSock, -1),
```

if it does crash, the handoff listener, for example, would be restarted and all would be fine. but previously we ignored the error, so just want to make sure this semantic change is ok.